modulefiles/tasks/hera/fire_emission.local.lua

```diff
@@ -1,3 +1,3 @@
 load("hpss")
-load("python_srw_cmaq")
-load(pathJoin("nco", os.getenv("nco_ver") or "4.9.3"))
+load("python_srw_aqm")
+load(pathJoin("nco", os.getenv("nco_ver") or "5.1.6"))
```

Comment:
Should this be 5.0.6?

```suggestion
load(pathJoin("nco", os.getenv("nco_ver") or "5.0.6"))
```